src/test/java/testsuite/regression/ConnectionRegressionTest.java

```diff
@@ -11757,6 +11765,7 @@ public void testBug99076() throws Exception {
      */
     @Test
     public void testBug98667() throws Exception {
+        final int oldLoginTimeout = DriverManager.getLoginTimeout();
```

Comment:
Do we need this?

Reply:
Not needed, just at the time thought it was weird that timeout was set without having it reset to what it was originally. Checking again shows that the original was also just `0`.

Reply:
If we are worried about it I suggest we add it in `beforeEach` and `afterEach`